README.md

```diff
@@ -21,4 +23,31 @@ honcho start
  
  :exclamation:  
  Using Gunicorn is not required, you can use any WSGI HTTP Server.  
- Using Honcho is not required, you can run processes listed in `Procfile` with another tool (e.g. Supervisor).
\ No newline at end of file
+ Using Honcho is not required, you can run processes listed in `Procfile` with another tool (e.g. Supervisor).
+ 
+### Run with docker
+ 
+#### docker-compose
+```
+git clone https://github.com/superdesk/video-server-app.git
+docker-compose up -d
```

Comment:
I think you have to do `cd video-server-app` in between

Reply:
right